src/test/shared/logger/outputChannelLogger.test.ts

```diff
@@ -65,6 +65,7 @@ describe('OutputChannelTransport', async function () {
                     level: 'info',
                     message: text,
                     [MESSAGE]: text,
+                    raw: false,
```

Comment:
can we add a test in this file, by adding a `raw` parameter to `runTests()` ?

Reply:
Yeah working on it.